test/setup.js

```diff
@@ -0,0 +1 @@
+require('mocha-suppress-logs')();
```

Comment:
I was not able to locate where this file is being used. What is the purpose of this file?

Reply:
This file is also purely for testing purposes?

Reply:
ok.